Examples/src/main/kotlin/com/mapbox/vision/examples/poi/POI.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.vision.examples.poi
+
+import android.graphics.Bitmap
+
+data class POI(val longitude: Double, val latitude: Double, val bitmap: Bitmap)
```

Comment:
I'd either put those classes to POIActivityKt.kt itself or created separate module for this examples.

Reply:
First option will work with the current scheme of the project, where each example is single activity, so there is no confusion between files/packages.
Second will allow building more complex examples, where putting all classes into single activity won't look nice. 
Thinking about this more - seems second approach is better. At the same time I planned to implement it in my own PR - so no work needed here, I'll implement it myself

Reply:
I've thought about this too and now think it is better to put data classes to the single activity rather than keep in separate package. It is kind of unusual pattern but I don't see big troubles for understanding with some additional simple data classes inside single Activity.

@yunikkk what do you mean about own PR? I would like to move auxiliary data classes to POIActivity

Reply:
I'll create PR with different example (USB camera) soon, and I've thought about splitting activities into modules to ease understanding of each example. 
At the same time I agree that adding this auxiliary classes to the activity itself is a good idea since they are not used anywhere else

Reply:
fixed